### PR TITLE
Adding testgres

### PR DESCRIPTION
**Pull request by demonolock**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by demonolock** [comment]
This is ready for review @conda-forge/staged-recipes

File: LICENSE.txt
Comment (xhochy) [inline LICENSE.txt:1]: Please don't delete this file.

File: recipes/testgres/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - setuptools
+  run:
+    - python
+    - pg8000
+    - port-for >=0.4
+    - six >=1.9.0
+    - psutil
+    - packaging
+    - enum34  # [py<34]
+    - ipaddress  # [py<33]
```
Comment (xhochy) [inline recipes/testgres/meta.yaml:30]:
```suggestion
```

File: recipes/testgres/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/postgrespro/testgres/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5cec69ffbd4ec37d3713209326ae159f7a79c2c4dadd874b4f55f144d5962b17
+
+
+build:
+  number: 0
```
Comment (xhochy) [inline recipes/testgres/meta.yaml:14]:
```suggestion
  number: 0
  noarch: python 
```

File: recipes/testgres/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
```
Comment (xhochy) [inline recipes/testgres/meta.yaml:19]:
```suggestion
    - python >=3.6
```

File: recipes/testgres/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - pip
+    - setuptools
+  run:
+    - python
```
Comment (xhochy) [inline recipes/testgres/meta.yaml:23]:
```suggestion
    - python >=3.6
```

File: recipes/testgres/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    - pg8000
+    - port-for >=0.4
+    - six >=1.9.0
+    - psutil
+    - packaging
+    - enum34  # [py<34]
+    - ipaddress  # [py<33]
+
+test:
```
Comment (xhochy) [inline recipes/testgres/meta.yaml:32]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check 
```

**Review by xhochy — CHANGES_REQUESTED** [review]
Let's make this noarch. This will simplify maintenance a lot.

**Review by xhochy — CHANGES_REQUESTED** [review]
Let's make this noarch. This will simplify maintenance a lot.

File: LICENSE.txt
Comment (demonolock) [reply LICENSE.txt:1]: removing the file reverted

**Review by xhochy — APPROVED** [review]

File: LICENSE [hunk 6]
```diff
@@ -0,0 +1,9 @@
+testgres is released under the PostgreSQL License, a liberal Open Source license, similar to the BSD or MIT licenses.
```
Comment (ocefpaf) [inline LICENSE:1]: This file should be removed, no? It is in the root directory.

**Pull request merged**